storage/rocksdb/build_rocksdb.cmake

```diff
@@ -77,11 +92,17 @@ if(ZSTD_FOUND AND (NOT WITH_ROCKSDB_ZSTD STREQUAL "OFF"))
     add_definitions(-DZSTD)
     include_directories(${ZSTD_INCLUDE_DIR})
     list(APPEND THIRDPARTY_LIBS ${ZSTD_LIBRARY})
+    add_feature_info(ROCKSDB_ZSTD "ON" "Zstandard Compression in the RocksDB storage engine")
+  else()
+    add_feature_info(ROCKSDB_ZSTD "OFF" "Zstandard Compression in the RocksDB storage engine")
   endif()
+else()
+  add_feature_info(ROCKSDB_ZSTD "OFF" "Zstandard Compression in the RocksDB storage engine")
 endif()
 
 add_definitions(-DZLIB)
 list(APPEND THIRDPARTY_LIBS ${ZLIB_LIBRARY})
+add_feature_info(ROCKSDB_ZLIB "ON" "zlib Compression in the RocksDB storage engine")
 
 if(CMAKE_SYSTEM_NAME MATCHES "Cygwin")
   add_definitions(-fno-builtin-memcmp -DCYGWIN)
```

Comment:
Samew thing, repetition for ROCKSDB_XXX compression features. maybe one add_feature_info per algorithm  would suffice.